modules/ROOT/pages/_partials/gs-writing-walkthroughs-proc.adoc

```diff
@@ -67,29 +67,24 @@ cd walkthrough-template
 
 
 === Understanding the file structure
-
 [source]
 ----
-.
 ├── walkthroughs-config.json <1>
-└── walkthroughs<2>
-    ├── <identifier> <3>
-    │   ├── images <4>
-    │   │   └── image.png
-    │   ├── walkthrough.adoc <5>
-    │   └── walkthrough.json <6>
-    ├── <walkthrough2-id> <3>
-    └── <walkthrough3-id> <3>
+├── walkthroughs <2>
+│   └── 1-template-walkthrough <3>
+│   ├── images <4>
+│   │   └── image.png
+│   ├── walkthrough.adoc <5>
+│   └── walkthrough.json <6>
 ----
 
-
 <1> This is an optional file that defines metadata for the set of Solution Patterns in the repository.
 
 <2> This is a required directory. All {walkthrough-name}s are defined under `walkthroughs`.
 
 <3> The directory name of a {walkthrough-name}. This is also displayed in the URL of {webapp}.
 
-<4> All images for the {walkthrough-name}.
+<4> All images for the {walkthrough-name}. This directory must be manually created by the user.
```

Comment:
I think this is a bit passive. 

Maybe... The user must manually create the directory